e2e/tools/validator/src/validator/config/__init__.py

```diff
@@ -50,49 +60,57 @@ def load(config_file: str) -> Validator:
     Returns:
         Config: A named tuple containing the configuration values.
     """
-    with open(config_file, 'r') as file:
+    with open(config_file, "r") as file:
         config = yaml.safe_load(file)
 
-    remote_config = config['remote']
+    remote_config = config["remote"]
     # NOTE: set default path to pkey if password is not set
 
-    pkey = remote_config.get('pkey', '')
-    if pkey != '':
-        pkey=os.path.expanduser(pkey)
+    pkey = remote_config.get("pkey", "")
+    if pkey != "":
```

Comment:
in case of 
```
remote:
  host: xxx
  username: xx
  password: xxx
  pkey:
```
this check fails and tries to connect with `pkey` Shouldn't this be `if pkey is not None`?

Reply:
good find!

Reply:
fixed now 🤞